### PR TITLE
Hello Microverse project

### PR DESCRIPTION
**Pull request by mntedev**
In this project 
   added .github/workflows folder.  
   added linters configuration yml file for HTML and CSS.
   added index.html, then create  " Hello microverse" header text.
   added style.css for index.html and style hello microverse header text.
   created README file for the project.
   created .gitignore file to ignore node_modules and test.md.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <h1 class="header"> Hello Microverse</h1>
+</body>
```
Comment (m05e5) [inline index.html]: - [ ] Please as part of the requirement, its better to put your "< h1 >" tag in a "< header >" tag for your code to be more semantic and for SEO

**Review by m05e5 — CHANGES_REQUESTED** [review]
Hi @worashf , 

Good job so far. I really like what you did!
There are some minor issues that you still need to work on to go to the next project but you are almost there!
mario_ladders

## Good points 👍 
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️

## Highlights


## Required Changes ♻️
* You really did a great job but there are some features you need to improve
  
  - [ ] Please as part of the requirement, its better to put your "< h1 >" tag in a "< header >" tag for your code to be more semantic and for SEO


_I'm sure you can do it just keep it up_
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_@m05e5_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (mntedev) [reply index.html]: included  header text in header tag

**Review by smunozmo — APPROVED** [review]
Hi @worashf, 👋 

## Highlights 👏 

- Great work using the right GitHub Flow.
- You have a professional-looking README, well done!
- Commits are descriptive, good work.
- Your linters are working properly.

Your project is complete!  There is nothing else to say other than... it's time to merge it! 🚀 
Very nice work, congratulations! 🎉 

### APPROVED! 👏 👏 👏 



Cheers and Happy coding! 🤓 

✅  Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by smunozmo — APPROVED** [review]
Your project is complete, great job! 👏

**Pull request merged**